sql/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -2581,7 +2583,7 @@ class SparkConnectPlanner(
     val result = SqlCommandResult.newBuilder()
     // Only filled when isCommand
     val metrics = ExecutePlanResponse.Metrics.newBuilder()
-    if (isCommand) {
+    if (isCommand || isSqlScript) {
```

Comment:
Do we have a test for SQL script execution using Spark Connect?

Reply:
Not yet. Is there an appropriate existing suite to add test there?

Reply:
Since you state (in PR's title) that this PR supports SQL scripting even for Spark connect, we need a test for that. @grundprinzip Where would you recommend to place the test?